zilliqa/src/cfg.rs

```diff
@@ -58,7 +62,9 @@ impl Default for NodeConfig {
             allowed_timestamp_skew: Duration::from_secs(10),
             data_dir: None,
             consensus_timeout: Duration::from_secs(5),
+            tx_retries: 1000,
             genesis_committee: vec![],
+            genesis_balance_each: 10000000000000000000,
```

Comment:
I would say this is fine, because the default genesis accounts is empty, and so nothing will get funded in the end. What do you think? It might conflict with James' PR anyway.

Reply:
Sure, that's a good point